test/new-e2e/pkg/environments/kubernetes.go

```diff
@@ -6,11 +6,14 @@
 package environments
 
 import (
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
 )
 
 // Kubernetes is an environment that contains a Kubernetes cluster, the Agent and a FakeIntake.
 type Kubernetes struct {
+	AwsEnvironment *aws.Environment
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You should use the new interface to pass the environment through the kubernetes env, passing it here makes `environments.Kubernetes` strictly bound to aws. An alternative is to define a custom environment `AwsKubernetes`

Reply:
I will move to the new interface later, as there are lots of change involved, outside of the scope of this PR